internal/cli/cmd/install/install.go

```diff
@@ -25,15 +25,32 @@ func NewCmd() *cobra.Command {
 	o := install.NewInstallOptions()
 	logger := cmdUtil.GetDefaultLoggerInstance()
 	cmd := &cobra.Command{
-		Use:     "install <components>",
-		Short:   "Command for ob-operator and components installation",
+		Use:   "install <component>",
+		Short: "Command for ob-operator and components installation",
+		Long: `Command for ob-operator and components installation.
+
+Currently support:
+- ob-operator: A Kubernetes operator that simplifies the deployment and management of OceanBase cluster and related resources on Kubernetes.
+- ob-dashboard: A web application that provides resource management capabilities.
+- local-path-provisioner: Provides a way for the Kubernetes users to utilize the local storage in each node, Storage of OceanBase cluster relies on it, which should be installed beforehand.
+- cert-manager: Creates TLS certificates for workloads in Kubernetes and renews the certificates before they expire, ob-operator relies on it for certificate management, which should be installed beforehand.
+
+if not specified, install all the components`,
 		PreRunE: o.Parse,
 		Args:    cobra.MaximumNArgs(1),
 		Run: func(cmd *cobra.Command, args []string) {
-			if err := o.Install(); err != nil {
-				logger.Fatalln(err)
+			if len(args) == 0 {
+				logger.Println("Install all the components")
```

Comment:
I think it's better to just install the ob-operator and ob-dashboard by default, local-path-provisioner and cert-manager are both 3rd party dependencies, these may already exist on the user's environment.

Reply:
That's true. `local-path-provisioner` is a weak dependency, and you could check the installation of `cert-manager` by checking existence of resources in `cert-manager.io/v1` like following,

<img width="1050" alt="image" src="https://github.com/user-attachments/assets/dc823de0-f47c-4638-bb62-270d6d597357">

Reply:
Ok,  Next, I will only provide installation of ob related components by default.